test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -284,7 +284,8 @@ def test_conv2d_with_quantizer_api(self):
                 # one for input and weight of the conv, one for output for the conv
                 torch.ops.quantized_decomposed.quantize_per_tensor.default: 2,
                 torch.ops.quantized_decomposed.dequantize_per_tensor.default: 2,
-                torch.ops.quantized_decomposed.quantize_per_channel.default: 1,
+                # note: quantize op for weights are const propagated
+                torch.ops.quantized_decomposed.quantize_per_channel.default: 0,
```

Comment:
Hi @jerryzh168, another question is looks like `fold_quantize` doesn't turn on by default, why will these number got changed?

Reply:
this is the long term plan, so I'm setting this flag to True in x86 tests as well, I'll update the PR soon.

Reply:
forgot to test before updating the current commit

Reply:
Got it. Thanks for the detail explanation.